### DISCUSSION TITLE
How to disable next telemetry

### DISCUSSION THREAD

**Discussion by rburgst** [keystonejs/keystone#9241] [Questions] [answered]
currently the keystone build logs that next.js telemetry is being collected. As only keystone is instrumenting the next build, I am unsure about how to turn it off.

**Comment by kennedybaird** [comment, marked as answer]
Per the [Next docs](https://nextjs.org/telemetry):

### env variables:
`NEXT_TELEMETRY_DISABLED=1` seems to work immediately.

To verify:
1. I set `NEXT_TELEMETRY_DEBUG=1`
2. ran `yarn dev`
3. saw a `[telemetry]` object logged (after `✨ Preparing Admin UI app`)
4. I set the `NEXT_TELEMETRY_DISABLED=1`  var
5. ran `yarn dev` again, no more log.

_I also tried with `yarn next telemetry disable`, it confirmed it disabled it, but when I ran `yarn dev`, I still received the telemetry log - so go with the `.env` option._

**Comment by dcousens** [comment]
Please see https://nextjs.org/telemetry

> You may also opt-out by setting an environment variable: NEXT_TELEMETRY_DISABLED=1.
